<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(97, 53, 131);
alternate-background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
color: rgb(42, 8, 89);
gridline-color: rgb(97, 53, 131);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextBrowser" name="chatBrowser">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(110, 100, 191, 28), stop:1 rgba(255, 255, 255, 255));
border-color: rgba(72, 31, 167, 0);
color: rgb(42, 8, 89);
border-color: rgba(191, 64, 64, 0);</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: italic 11pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>Message:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="messageLineEdit">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(93, 6, 66);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendMessageButton">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(97, 53, 131);
background-color: rgba(99, 15, 142, 46);
border-width: 3px;        
border-style: solid;
border-radius: 9px;
margin:9px;
padding:9px;
qproperty-icon: &quot;:/image/message_icon.qrc&quot;;</string>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
        <property name="autoRepeatInterval">
         <number>99</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain_menu">
    <property name="title">
     <string>Main menu</string>
    </property>
    <addaction name="actionDeleteAccount"/>
    <addaction name="separator"/>
    <addaction name="actionCloseUser"/>
   </widget>
   <addaction name="menuMain_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDeleteAccount">
   <property name="text">
    <string>Delete Account</string>
   </property>
  </action>
  <action name="actionCloseUser">
   <property name="text">
    <string>Close Client</string>
   </property>
   <property name="toolTip">
    <string>Close user</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
